src/java.base/share/classes/java/lang/SecurityManager.java

```diff
@@ -313,7 +312,12 @@
  * @see     java.security.ProtectionDomain
  *
  * @since   1.0
+ * @deprecated The Security Manager is deprecated and subject to removal in a
```

Comment:
Javadoc will prefix, in bold, "Deprecated, for removal: This API element is subject to removal in a future version". I'm just wondering if the sentence will be repeated here, can you check?

Reply:
It includes both:
![Screen Shot 2021-05-18 at 8 41 11 AM](https://user-images.githubusercontent.com/35072269/118652730-dfb35400-b7b4-11eb-83ee-92be9136fea2.jpg)

Reply:
Thanks for checking, I assumed that was the case so wondering if it should be dropped from the deprecation text to avoid the repeated sentence.

Reply:
My feeling is that it is better to be more specific than the generic removal text as this is the most significant class that is being deprecated for removal. Also, this says "the Security Manager" (note the space between) which is really encompasses more than the `java.lang.SecurityManager` API and which is explained more completely in the JEP.